var/spack/repos/builtin/packages/ds/package.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+from os import symlink
+
+
+class Ds(AutotoolsPackage):
+    """SAOImage DS9 is an astronomical imaging and data visualization
+       application."""
+
+    homepage = "http://ds9.si.edu/"
+    url      = "http://ds9.si.edu/download/source/ds9.8.0rc6.tar.gz"
```

Comment:
The URL is for 9.8.0rc6 but the version is for 9.8.0. Can you make them consistent?

Reply:
updated accordingly.